posts/2023/09/26/new-ansible-galaxy.md

```diff
@@ -0,0 +1,100 @@
+
+It's been a long road to get here. We have had to continue with the progression of an automation hub at the same time as delivering community-only features into the code base, likesocial authentication for GitHub being required in Galaxy but not in the automation hub. 
+
+We are now at the point where we should make the switch and start the new journey on the [galaxy_ng](https://github.com/ansible/galaxy_ng/) code base, together. 
+
+ 
+
+## The Switch
+The new galaxy service has been running since the beginning of the year on a URL known as beta-galaxy.ansible.com. This URL has hosted many community members and some institutions, who have helped interact with us on the service. The old galaxy.ansible.com service has been running as per usual; we have a sync between the two to make sure the beta-galaxy.ansible.com site is up to date.
+
+We already have a URL called old-galaxy.ansible.com pointing to a read-only version of the galaxy.ansible.com service as it is today.
+
+On September 30, we will switch the IP records used by galaxy.ansible.com to use the records that beta-galaxy.ansible.com uses. 
+
+* Summary: https://old-galaxy.ansible.com/ will point to old Galaxy
+* https://galaxy.ansible.com/ will point to new [galaxy_ng](https://github.com/ansible/galaxy_ng/)
+* https://beta-galaxy.ansible.com/ will point to new [galaxy_ng](https://github.com/ansible/galaxy_ng/)
+
+This means if the switch is not successful, we can easily switch back or ask the community to try the older service. But we have tested a lot, so everything should go to plan.
+
+ 
+
+## After the Switch
+@rochacbruno has written up this awesome [table](https://github.com/ansible/galaxy_ng/discussions/1729) to show what was in Galaxy vs the new beta Galaxy site. You will notice not everything has been reimplemented into [galaxy_ng](https://github.com/ansible/galaxy_ng/) due to time constraints, but we are open to discussions on [forum.ansible.com](https://forum.ansible.com) if something should come back, and more importantly, we wish for the community to help design new features to be introduced. 
+
+A good example of features that have not yet been reformulated is the current scoring system.We are retaining the old scores, so nobody loses out on the kudos they have earned already, however, we feel that a new scoring system needs to be implemented,and we’d love to begin soliciting community contributions for how we can improve it. 
+
+| Feature                 | Galaxy  | Beta Galaxy | Reason    |
+|-------------------------|-------- |-------------|-----------|
+| Sign in/up via GitHub   | ✔️     | ✔️         |           |
+| API Key                 | ✔️     | ✔️         |           |
+| E-mail/App Notifications | ✔️     | ❌          | Low usage |
```

Comment:
```suggestion
| E-mail/App Notifications | Available     | Not available          | Low usage |
```

Reply:
Let's replace the tickmark and x with something more meaningful, please. This would help with accessibility. While I understand this constitutes a change from the original I believe it's worthwhile.

Reply:
Right, it absolutely worth it, why write if you are not writing for accessibility?! I will change them

Reply:
@oraNod what should be used in place of the tickmark and x? do you have a suggestion I would love to work on this during the weekend.

Reply:
Committing your suggestion and implementing it across the table.